ajax_select/__init__.py

```diff
@@ -13,6 +13,9 @@
 from django.utils.encoding import force_text
 from django.utils.html import escape
 from django.utils.translation import ugettext_lazy as _
+from django.utils.module_loading import autodiscover_modules
```

Comment:
Okay, I addressed your feedback and Django 1.5/1.6 is now supported with the same style of lookup channel configuration in the settings.py, while both settings.py and autodiscovery are available for Django 1.7+, with the only restriction being that you can't use the same label for a lookup channel more than once.  I kept settings.py configuration even for Django 1.7+, because simple lookup channel registration still needs to be supported in settings.py with the "{'person'  : {'model': 'example.person', 'search_field': 'name'}" style.  The version sniffing is done at the bottom of the __init__.py in the ajax_select app root.  Oh, and I went through the flake style errors and corrected those as well.  How does this look to you?